### DISCUSSION TITLE
Glyphs being copied like unicode

### DISCUSSION THREAD

**Discussion by bwkam** [polybar/polybar#2825] [General]
Hello, so I installed Fonts Awesome 5 manually, and did a fc-cache, but then when I copy the icons off the Fonts Awesome page into Neovim, they are copied like this;
![image](https://user-images.githubusercontent.com/91009118/192296616-4194f74c-9f16-4dee-aaa8-baac3cd0afd9.png)
They are rendered the same in the bar.
That's how I included the fonts;
![image](https://user-images.githubusercontent.com/91009118/192296894-55390745-76fe-4eda-a9c5-7cf7dc960621.png)
Distro: Arch

**Comment by patrick96** [comment]
Polybar doesn't support unicode escape codes, you need to copy the actualy character. The font awesome does have a "Copy Glyph" button which does this for you.

**Reply by bwkam** [reply]
> Polybar doesn't support unicode escape codes, you need to copy the actualy character. The font awesome does have a "Copy Glyph" button which does this for you.

Yeah I'm sure I did copy the glyph using this copy glyph button, but that's how they get copied.

**Reply by patrick96** [reply]
Then this is maybe a neovim issue. You can confirm that if pasting in other places (other editor, your browser, ...) works correctly.
